src/viam/sdk/module/service.hpp

```diff
@@ -56,5 +58,22 @@ class ModuleService_ : public viam::module::v1::ModuleService::Service {
     std::string parent_addr_;
 };
 
+/**
+ * C++ modules must handle SIGINT and SIGTERM. Make sure to create a sigset
+ * for SIGINT and SIGTERM that can be later awaited in a thread that cleanly
+ * shuts down your module. pthread_sigmask should be called near the start of
+ * main so that later threads inherit the mask. SignalManager can handle the
+ * boilerplate of this functionality.
+ */
+class SignalManager {
+   public:
+    SignalManager();
+    /** wait on sigset */
```

Comment:
(very much a nit) just for consistency, can we indicate comments with `//` rather than `/**`?

Reply:
Yep!